src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -295,7 +295,7 @@
                 <li data-faq-entity-id="{{ faqParagraph.entity.entityId }}">
                   <button aria-controls="{{ faqParagraph.entity.entityBundle }}-{{ faqParagraph.entity.entityId }}" class="usa-accordion-button usa-button-unstyled" aria-expanded="false" type="button">{{ faqParagraph.entity.fieldQuestion }}</button>
                   <div id="{{ faqParagraph.entity.entityBundle }}-{{ faqParagraph.entity.entityId }}" class="usa-accordion-content" aria-hidden="true">
-                    {% assign fieldAnswer = faqParagraph.entity.fieldAnswer[0] %}
+                    {% assign fieldAnswer = faqParagraph.entity.fieldAnswer | first %}
```

Comment:
hahahaha seriously?? I had wondered

Reply:
Lol yeahhh you can also use `faqParagraph.entity.fieldAnswer.0` but it doesn't accept the square brackets